src/main/java/com/google/devtools/build/lib/actions/ResourceManager.java

```diff
@@ -137,6 +141,10 @@ public static ResourceManager instance() {
   // definition in the ResourceSet class.
   private double usedRam;
 
+  // Used amount of extra resources. Corresponds to the extra resource
+  // definition in the ResourceSet class.
+  private Map<String, Float> usedExtraResources;
```

Comment:
Could you please sync the changes from the repository. This file had significant changes during last month.